cmd/variable.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/azure/spin-aks-plugin/pkg/azure"
+	"github.com/azure/spin-aks-plugin/pkg/logger"
+	"github.com/spf13/cobra"
+)
+
+var kvName string
+var secretName string
+
+func init() {
+	rootCmd.AddCommand(variableCmd)
+	rootCmd.Flags().StringVar(&kvName, "kv-name", "", "Name of the keyvault to use")
```

Comment:
are these required? seems like something we should mark as required

Reply:
yep